### DISCUSSION TITLE
Kata 2:( Ayuda

### DISCUSSION THREAD

**Discussion by ValeriaGutierrezA** [LaunchX-InnovaccionVirtual/CursoIntroPython#278] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/99141921/154174867-caeb1234-4563-4556-890c-ecd659c95d42.png)
Alguien sabe como habilitar la ejecución de Scripts o qué tengo que hacer, por favor? Ya ando desesperada :(

**Comment by ArielCanedo** [comment]
Hola Vale
Primero tienes que crear el entorno virtual
Pruébalo en cmd

python -m venv env

Luego lo activas con

env\scripts\activate

**Reply by ValeriaGutierrezA** [reply]
Listo :3, ya me salió, muchas gracias! Pero no hay ningún problema si aparece en cmd y no en powershell?

**Reply by AlejandroMontoyaT** [reply]
No

**Comment by OscarPedroza** [comment, marked as answer]
Hola, intenta ejecutar los comandos desde una terminal de cmd

Desde este icono de más puedes agregarla en tu vs code, dandole a la flecha y seleccionando 'command promt'
![image](https://user-images.githubusercontent.com/83786890/154176235-fa3a8a0a-dd68-430e-95f6-63007c02b9d7.png)

**Reply by ValeriaGutierrezA** [reply]
Muchas gracias! Ya me salió

**Reply by AlejandroPedrazaR** [reply]
Muchas gracias Oscar, también estaba atorado con eso!
